crates/typst-kit/src/package.rs

```diff
@@ -109,6 +123,8 @@ impl PackageStorage {
             // version.
             self.download_index()?
                 .iter()
+                .lazy_deser::<MinimalPackageInfo>()
+                .filter_map(|res| res.ok())
```

Comment:
I wonder whether the `LazyDeser` infrastructure really pulls its weight compared to just having
```rust
.filter_map(|value| {
    serde_json::from_value::<MinimalPackageInfo>(value.clone()).ok()
})
```
inline here.

Reply:
Update: We can actually use `MinimalPackageInfo::deserialize(value).ok()` to get rid of the `.clone()`